CommandHelper/LocalPackages/commands/buffs/doctormanhattan.ms

```diff
@@ -10,5 +10,17 @@
             'op': 'true',
             'speech': "The existence of life is a highly overrated phenonenon.",
             'powers': array(),
+                "/kill",
+                "/gmc",
+                "/gms",
+                "/tp",
+                "/fly",
+                "/flyspeed <number>"
+            ),
             'buffs': array()
-        )
+                "/effect ". @plr ." 5 9999 25 true",
+                "/effect ". @plr ." 10 9999 4 true",
+                "/effect ". @plr ." 11 9999 5 true",
+                "/effect ". @plr ." 21 9999 14 true"             
```

Comment:
EVIDENCE REQUIRED: this can be set to basically anything because resistance 4 makes him invincible anyway